UKHomeOffice/sgmr-service
# SPCR-546/SPCR-550 Error when arrival and departure dates are identical

### AC / Description of changes
As a temporary solution to the current errors we see in live when users enter the same port for departure and arrival, implement a validation on exiting the arrival page to display an error if both the departure and arrival ports are the same.

ACs

Given user is on the Arrival page and entered the arrival port as same as the departure port   
When click on ‘Save and continue’ or ‘Save and come back later’ 
Then give an error message that they can’t have both arrival and departure ports the same.
And stop being able to save the report

### To Test
- Create a voyage plan
- Fill in the departure page - enter Portsmouth (PME)
- Continue to the arrival page
- Fill in the arrival page - enter Portsmouth (PME)
- Click continue
- > The form should error 'Your arrival point must not be the same as your departure point'
- Enter Portsmouth and choose one with a different UNLOCODE 
- Click save
- > You should be taken to the PC page

### Notes

---

**Remember to merge to feature branches not `master`.**
